.github/workflows/regression-report.yml

```diff
@@ -56,8 +63,22 @@ jobs:
               content: 'rocket'
             })
 
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - name: Check PR status
+        id: check_pr_status
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_URL: ${{ github.event.issue.pull_request.url }}
+        run: |
+          PR_STATE=$(curl --fail-with-body -s -H "Authorization: token $GITHUB_TOKEN" \
+                     -H "Accept: application/vnd.github.v3+json" \
+                     "$PR_URL" | jq -r .state)
+          if [ "$PR_STATE" = "open" ]; then
+            echo "PR is active"
+            ./.ci/append-to-github-output.sh "is_active" "true"
```

Comment:
Why we need this output production? Can we simply fail workflow right after curl response checking?

Reply:
@Romani, i have made changes, please let me know your feedback.